power.c

```diff
@@ -0,0 +1,21 @@
+//Emma L
+//powers below n^2
+
+
+#include <stdio.h>
+#include <math.h>
+int main(){
+	printf("squares numbers under 1000\n");
+
+	int x;
+
+	x=0;
+	int square;
+	while(square<1000){
+		square = pow(2,x);
```

Comment:
The order of your print and square variable is incorrect - because you put the print statement after, it printed one too many numbers.